about.html

```diff
@@ -16,8 +16,14 @@
         <div class="header-top">
             <div class="container row">
```

Comment:
- Hey, to enforce good use of the semantic tag, kindly change this div to nav tag as the div is mainly used to divide different parts of the webpage and as such, it has no semantic meaning. the nav tag would be the correct tag to use here as this contains the introductory content of your page which in your case is best applicable to be used here. please, update this tag to improve your semantic usage. :pray: